libraries/bxdf/standard_surface.mtlx

```diff
@@ -19,33 +19,33 @@
             doc="Color tint on the specular reflection." />
       <input name="specular_roughness" type="float" value="0.2" uimin="0.0" uimax="1.0" uiname="Specular Roughness" uifolder="Specular"
             doc="The roughness of the specular reflection. Lower numbers produce sharper reflections, higher numbers produce blurrier reflections." />
-      <input name="specular_IOR" type="float" value="1.5" uimin="0.0" uimax="3.0" uiname="Index of Refraction" uifolder="Specular"
+      <input name="specular_IOR" type="float" value="1.5" uimin="0.0" uisoftmax="3.0" uiname="Index of Refraction" uifolder="Specular"
             doc="Index of refraction for specular reflection." />
       <input name="specular_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Anisotropy" uifolder="Specular" uiadvanced="true"
             doc="The directional bias of reflected and transmitted light resulting in materials appearing rougher or glossier in certain directions." />
-      <input name="specular_rotation" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
+      <input name="specular_rotation" type="float" value="0" uisoftmin="0.0" uisoftmax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
```

Comment:
Rotation should stay in [0, 1] "hard" range. This is internally mapped to radians where 1.0 gives full circle, so there is never a need to go higher than 1.0.